graphql_compiler/compiler/emit_sql.py

```diff
@@ -731,6 +763,36 @@ def traverse(self, vertex_field, optional):
         self._relocate(self._current_location.navigate_to_subpath(vertex_field))
         self._join_to_parent_location(previous_alias, edge.from_column, edge.to_column, optional)
 
+    def _wrap_into_cte(self):
```

Comment:
missing type hints throughout this function -- let's make sure all new code is type-hinted